src/Element/MineOrganisationsData.php

```diff
@@ -21,14 +21,13 @@ public static function getCompositeElements(array $element): array {
     $elements = [];
 
     $elements['search'] = [
-      '#type' => 'fieldset',
-      '#title' => t('Search'),
+      '#type' => 'markup',
```

Comment:
This type is superfluous and maybe even invalid (cf. https://www.drupal.org/node/2036237).

```suggestion
```

Reply:
It is indeed superfluous. 
Not specifying a type would cause warnings (Undefined array key) in https://github.com/itk-dev/os2forms_selvbetjening/blob/develop/web/modules/custom/os2forms_selvbetjening/os2forms_selvbetjening.module#L26C8-L26C8, but this could obviously also be fixed.